integration-tests/src/main/java/org/apache/polaris/service/it/test/CatalogFederationIntegrationTest.java

```diff
@@ -159,6 +159,7 @@ private void setupCatalogs() {
         AwsStorageConfigInfo.builder()
             .setStorageType(StorageConfigInfo.StorageTypeEnum.S3)
             .setPathStyleAccess(true)
+            .setUserTokenSts(false)
```

Comment:
Is this required? Could we handle defaults transparently to code areas not affected by the new feature?